app/main.py

```diff
@@ -1 +1,38 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(
+            self,
+            name: str,
+            health: int = 100,
+            hidden: bool = False
+    ) -> None:
+        self.health = health
+        self.name = name
+        self.hidden = hidden
+        Animal.alive.append(self)
+
+    @classmethod
+    def death(cls) -> None:
+        for animal in cls.alive:
+            if animal.health <= 0:
+                cls.alive.remove(animal)
```

Comment:
remove the instance directly from the class attribute alive without using a `classmethod`, you can do so within the death method by checking the health of the instance and removing it if necessary.